backend/src/controllers/items-controller.ts

```diff
@@ -19,26 +21,21 @@ export const createItem = async (req: Request, res: Response) => {
     if (!dupeItem) {
         return res.status(409).json({error: "An item with the same name already exists."});
     }
-    //Checks if the ID matches the owner of the item
-    const itemOwnerId = req.body.itemOwnerId;
-    const itemOwnerExistence = await User.findOne({_id: itemOwnerId});
-    if (!itemOwnerExistence) {
-        return res.status(404).json({error: "This id does not match the owner of the item."});
-    }
     const name = req.body.name as string;
     const description = req.body.description as string;
     const pickupLocation = req.body.pickupLocation as string;
     const pickupTime = req.body.pickupTime as string;
-
-
-
+    //Checks if UserId exists
+    if(!req.token?.userid){
```

Comment:
This error occurs each time I make a call:
![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/896e5409-210a-4ecb-b059-1179b87c9a81)
It's because `userid` is actually `userId` (`userid` doesn't exist in the token)

```suggestion
    if(!req.token?.userId){
```

Fix this issue in the functions below as well (listed)